pb/rossby.proto

```diff
@@ -0,0 +1,81 @@
+syntax = "proto3";
+package pb;
+
+import "errors.proto";
+import "messages.proto";
+import "contacts.proto";
+
+// The Rossby service implements a gRPC API for sending encrypted messages.
+// The messages in this file describe the client-server interactions, not necessarily
+// the datastructures that are serialized in the backend or for replication.
+service Rossby {
+    rpc Register (RegisterRequest) returns (RegisterReply) {}
+    rpc Authorize (AuthorizeRequest) returns (AuthorizeReply) {}
+    rpc Contact (ContactRequest) returns (ContactReply) {}
+    rpc Fetch (FetchRequest) returns (Messages) {}
+    rpc Deliver (Messages) returns (DeliverResponse) {}
+
+    // TODO: is this a single bidirectional stream or two unidirectional streams?
+    rpc Chat (stream Messages) returns (stream Messages) {}
```

Comment:
Not 100% sure about how to get this to work the way we expect. If we do the following:

```
rpc Fetch (Status) returns (Messages) {}
rpc Deliver (Messages) returns (Status) {}
```

Then this RPC works very well, but I need to get the MVP with Fetch/Deliver done in order to learn if this will work or not.